package.json

```diff
@@ -130,5 +130,8 @@
   "peerDependencies": {
     "react": "^16.8.3",
     "react-dom": "^16.8.3"
+  },
+  "overrides": {
+    "jest-environment-node": "29.4.0"
```

Comment:
How did you find this?

Reply:
https://github.com/smooth-code/jest-puppeteer/issues/512, where there's an explanation about why they can't do it in their dependencies... but jest fixed things up stream so we don't need it any more either.